examples/2d/custom_gltf_vertex_attribute.rs

```diff
@@ -63,7 +59,13 @@ fn setup(
         Transform::from_scale(150.0 * Vec3::ONE),
     ));
 
-    commands.spawn(Camera2d);
+    commands.spawn((
+        Camera2d,
+        EnvironmentMapLight {
+            intensity: 1.0 / 5.0,
+            ..EnvironmentMapLight::solid_color(&mut images, Color::WHITE)
```

Comment:
Shouldn't we just be falling back to the default `DEFAULT_ENVIRONMENT_MAP_TEXTURE_HANDLE` for all the white examples?
```suggestion
            ..default()
```